templates/zerver/help/analytics.md

```diff
@@ -0,0 +1,35 @@
+# Analytics
+
+Zulip provides analytics for tracking various usage statistics about you and your organization.
+So, you can get a holistic view of how your organization's communications.
```

Comment:
Yeah!! i think i was going to write "how your organization communicates". I left the how when i changed the sentence. 
My bad!!